tests-utils/build.gradle

```diff
@@ -48,56 +36,38 @@ tasks.withType(ScalaCompile) {
+    api group: 'net.jodah',                      name: 'failsafe',                      version: '2.0.1'
+    api group: 'org.gradle',                     name: 'gradle-tooling-api',            version: '4.10.2'
+    api group: 'com.google.code.gson',           name: 'gson',                          version: '2.9.0'
+    api group: 'com.google.guava',               name: 'guava',                         version: '21.0'
+    api group: 'org.apache.httpcomponents',      name: 'httpclient',                    version: '4.5.13'
+    api group: 'org.apache.logging.log4j',       name: 'log4j-1.2-api',                 version: '2.15.0'
+    api group: 'org.apache.logging.log4j',       name: 'log4j-core',                    version: '2.15.0'
+    api group: 'io.jsonwebtoken',                name: 'jjwt-api',                      version: '0.11.2'
+    api group: 'io.jsonwebtoken',                name: 'jjwt-impl',                     version: '0.11.2'
+    api group: 'io.jsonwebtoken',                name: 'jjwt-orgjson',                  version: '0.11.2'
+    api group: 'com.jayway.jsonpath',            name: 'json-path',                     version: '2.5.0'
+    api group: 'io.monix',                       name: 'monix_2.12',                    version: '3.0.0-RC3'
+    api group: 'com.lihaoyi',                    name: 'os-lib_2.12',                   version: '0.7.0'
+    api group: 'eu.timepit',                     name: 'refined_2.12',                  version: '0.9.5'
+    api group: 'org.scala-lang',                 name: 'scala-library',                 version: '2.12.4'
+    api group: 'org.scala-lang.modules',         name: 'scala-java8-compat_2.12',       version: '0.9.0'
+    api group: 'com.typesafe.scala-logging',     name: 'scala-logging_2.12',            version: '3.9.2'
+    api group: 'org.scalatest',                  name: 'scalatest_2.12',                version: '3.1.0'
+    api group: 'com.dimafeng',                   name: 'testcontainers-scala_2.12',     version: '0.39.0'
+    api group: 'org.testcontainers',             name: 'testcontainers',                version: "1.15.0"
+    api group: 'eu.rekawek.toxiproxy',           name: 'toxiproxy-java',                version: "2.1.4"
+    api group: 'com.unboundid',                  name: 'unboundid-ldapsdk',             version: '6.0.6'
+    api group: 'com.mashape.unirest',            name: 'unirest-java',                  version: '1.4.9'
+    api group: 'com.lihaoyi',                    name: 'upickle_2.12',                  version: '0.7.1'
 
-license {
-    header rootProject.file('ReadonlyRESTLicenseHeader.txt')
-    skipExistingHeaders true
-    useDefaultMappings = false
-    mapping {
-        java = 'SLASHSTAR_STYLE'
-        scala = 'SLASHSTAR_STYLE'
-    }
-}
-
-githook {
-    hooks {
-        "pre-commit" {
-            shell = "./gradlew license --rerun-tasks"
-        }
-    }
 }
 
-task wrapper(type: Wrapper) {
+tasks.register('wrapper', Wrapper) {
```

Comment:
is it needed? I don't understand this task. Maybe we can try to remove it? (but please to it after solving all other issues)